Header header          #standard header information

VertexPointXYMsg estimate

geometry_msgs/Point position

std_msgs/Float64[] angle_width
std_msgs/Float64[] orientation
VertexSE2Msg first_seen_from